### ISSUE TITLE
[Problem/Bug]: WinUI3 `AreBrowserAcceleratorKeysEnabled=false` has no effect 

### ISSUE THREAD

**Issue by Eilon** [MicrosoftEdge/WebView2Feedback#4241]
### What happened?

Specifically in WinUI3, setting `AreBrowserAcceleratorKeysEnabled = false` has no effect. Keys such as <kbd>F5</kbd> continue working no matter what, even though they should stop working when the value is set to `false`.

The same functionality works as expected in WPF.

### Importance

Important. My app's user experience is significantly compromised.

### Runtime Channel

Stable release (WebView2 Runtime)

### Runtime Version

120.0.2210.61

### SDK Version

WinAppSDK 1.4.230913002

### Framework

WinUI3/WinAppSDK

### Operating System

Windows 11

### OS Version

22H2 22621.2861

### Repro steps

1. Create new WinUI3 Packaged Desktop app
2. Change XAML code to just `<WebView2 x:Name="WebView2Control" Source="https://www.bing.com" HorizontalAlignment="Stretch" VerticalAlignment="Stretch" />`
3. Change CS code to:
   ```c#
        public MainWindow()
        {
            this.InitializeComponent();
            WebView2Control.CoreWebView2Initialized += WebView2Control_CoreWebView2Initialized;
        }

        private void WebView2Control_CoreWebView2Initialized(WebView2 sender, CoreWebView2InitializedEventArgs args)
        {
            sender.CoreWebView2.Settings.AreBrowserAcceleratorKeysEnabled = false;
        }
   ```
4. Run app, and then press <kbd>F5</kbd>, <kbd>Ctrl</kbd>+<kbd>P</kbd> or any other typical browser shortcut

EXPECTED: These keys should be ignored

ACTUAL: The browser responds to these and refreshes/prints/etc.

NOTE: The same code in a WPF WebView2 app works as expected.

### Repros in Edge Browser

Not Applicable

### Regression

Don't know

**Comment by tofuandeve** [comment]
@gameDNAstudio  Hi, we're taking a quick look at this issue, could you provide some more information on the drag and drop scenario? Which API did you use?

**Comment by gameDNAstudio** [comment]
@tofuandeve sometimes when I click too fast on the WebView it starts drag & drop action of nothing (or text probably). Then it leads to very often to crash. 

I attached screenshot with showing the icon for this action.

![Web](https://github.com/MicrosoftEdge/WebView2Feedback/assets/15182229/5b4a59bd-88db-4221-86b5-cc382c9371aa)

**Comment by tofuandeve** [comment]
@gameDNAstudio  got it! And you were able to disable this before or was it always disabled by default and recently it was enabled after updating to later WebView2 runtime?

**Comment by gameDNAstudio** [comment]
@tofuandeve I didn't use WebView2 before so I don't know how it was before. Yes, I click within the WebView.

**Comment by tofuandeve** [comment]
@gameDNAstudio  after a quick look, the dragging issue seems to be on the WinUI side, we'll get the right folks to take a look at it. For now we'll continue to look at the issue with AreBrowserAcceleratorKeysEnabled setting. 

Thanks.

**Comment by tofuandeve** [comment]
This issue is tracked here https://github.com/MicrosoftEdge/WebView2Feedback/issues/4278#issuecomment-1878993354
This seems to be a regression in WebView2. Please refer to the above thread for future updates.
Thanks!

**Comment by victorhuangwq** [comment]
Closed issue as duplicate of #4278

**Comment by richardhauer** [comment]
According to #4278 this is supposedly fixed in Canary 122 but I have now the beta 122.0.2365 for Edge and the `AreBrowserAcceleratorKeysEnabled` setting still has no effect in a Blazor hybrid app in MAUI. Is there still an issue with WinUI3 here?
